exdb/views.py

```diff
@@ -107,34 +109,52 @@ def get_success_url(self):
 
     def get_context_data(self, **kwargs):
         context = super(ExperienceApprovalView, self).get_context_data()
-        context['experience'] = self.get_experience()
+        if kwargs.get('invalid_comment'):
```

Comment:
These lines (along with some in `form_valid()` & `form_invalid()`) could use some more documentation explaining exactly what's going on.